src/features/CodeLinkFeature.ts

```diff
@@ -559,3 +605,23 @@ async function resolveWorkspaceSymbol(
 	}
 	return undefined;
 }
+
+async function resolveCustomSymbol(
+	uri: Uri,
+	symbol: string
+): Promise<Range | undefined> {
+	const d = await workspace.openTextDocument(uri);
+	const e = await window.showTextDocument(d, {
+		viewColumn: ViewColumn.One,
+		preserveFocus: true,
+	});
+	const text = e.document.getText();
+	const index = text.indexOf(symbol);
+	if (index > -1) {
+		var startPos = e.document.positionAt(index);
+		var endPos = e.document.positionAt(index + symbol.length);
+		const range = new Range(startPos, endPos);
+		return range;
+	}
+	return undefined;
+}
```

Comment:
I'm not so happy with just using search to find the location.
It would only work for text that occurs once in a file.